docs/configuration.asciidoc

```diff
@@ -616,6 +616,22 @@ IMPORTANT: Use of `ServerUrls` is deprecated. Use `ServerUrl`.
 | `http://localhost:8200` | String
 |============
 
+[float]
+[[server-use-windows-credentials]]
+==== `ServerUseWindowsCredentials`
+
```

Comment:
Since basicly all it does is set the dotnet HttlClientHandler.UseDefaultCredentials property, maybe we can reuse some of the microsoft documentation for that: https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.usedefaultcredentials?view=net-6.0 

So maybe something like: 
"Set this property to true when requests made by the apm-agent should, if requested by the server, be authenticated using the credentials of the currently logged on user. This is useful when using windows authentication on a proxy, that routes apm-agent requests to the apm-server."

Reply:
Perfect, I like it! Let's go with this.

Reply:
added asciidoc description